cli/cmd/endpoints.go

```diff
@@ -59,6 +59,7 @@ func newCmdEndpoints() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "endpoints [flags]",
+		Aliases: []string{"ep"},
```

Comment:
@Pothulapati Thanks for tackling this! Since it addresses a separate issue from #2273, can I trouble you to put it in a separate PR?

Reply:
Yes, It was a mistake. I unexpectedly pushed it to the same branch.

Reply:
No prob! Am glad to see it's such a straightforward fix.

Reply:
Done :)